backend/config.yaml

```diff
@@ -0,0 +1,4 @@
+mysql_pass="password"
```

Comment:
These parameters should be passed by some `-var-file`. Find the solution in the next sprint